README.md

```diff
@@ -21,6 +21,13 @@ Config.php parameters to operate the server in master mode:
 
 // operation mode
 'gss.mode' => 'master',
+
+// define a master admins, this users will not be redirected to a slave but are
+// allowed to login at the master node to perform administration tasks
+'gss.master.admin' => ['admin1Uid', 'admin2Uid'],
+
+// define a parameter given by the IdP to decide where a user is located
+'gss.saml.slave.mapping' => 'idpParameterSlave',
```

Comment:
I'm unsure about this one. Often the operators of the NC and the SAML are different teams. Also requesting it from SAML is IMO the wrong approach. The globalsiteselector should have different configurable/scriptable strategies that are executed and then return where to place a user. Otherwise SAML needs to know which NC instances are available and where to put a user (how many users are on instances, how filled are they, ...).

I would rather allow a setting of a class that is then initialized and feed with all the data from SAML + maybe the lookup server and then this class can decide where to put the user. The default one should be a round-robin class that just randomly returns one of the available servers.

Reply:
This parameter is for initial user deployment. If the user login for the first time, letting saml parameter decide where to deploy the user is one option among others we discussed and agreed on. Of course we can't hardcode the name of the saml parameter, that's why there is a config switch.